Felienne/hedy
# [IMPROVEMENT] Improves readability "common mistakes" for teachers page

**Description**
In this PR we re-structure the for-teachers page to show the common mistakes levels as sections instead of one long page. These sections are automatically divided with buttons, which can be pressed to show each corresponding section.

**Fix for**
This PR fixes #1865

**How to test**
Make sure you are a teacher and navigate to the for-teachers page. When using the documentation notice the new structure. Verify that every button is working as expected.
